lib/fortnox/api/validators/edi_information.rb

```diff
@@ -0,0 +1,16 @@
+require "fortnox/api/validators/base"
+
+module Fortnox
+  module API
+    module Validator
+      class EDIInformation
+
+        extend Fortnox::API::Validator::Base
+
+        using_validations do
+          # No validations needed, in other words always valid.
```

Comment:
Isn't it better to implement this as a separate class then? Something that has the same API as a real validator instance but just statically returns `true` for the `validate()` call?